napari/utils/transforms/transforms.py

```diff
@@ -195,6 +203,7 @@ def __init__(self, scale=(1.0,), translate=(0.0,), *, name=None):
 
         self.scale = np.array(scale)
         self.translate = np.array(translate)
+        self.is_diagonal = True
```

Comment:
I guess it's OK to have different ways to implement `is_diagonal` (i.e. here a plain attribute, vs. a `@cached_property` in `Affine`), but it's a little confusing.